changelogs/fragments/125-api.yml

```diff
@@ -0,0 +1,4 @@
+minor_changes:
+  - api_modify, api_info - support API paths ``interface pppoe-client``, ``interface vlan``, ``interface bridge``, ``interface bridge vlan``
+    and also setting ``interface bridge port ingress-filtering`` field default to True whih is the default in ROS
```

Comment:
```suggestion
    and also setting ``interface bridge port ingress-filtering`` field default to ``true`` which is the default in ROS
```

Reply:
Can you move the `interface bridge port ingress-filtering` part to a `bugfixes:` section in this file? So that you have
```yaml
minor_changes:
  - ...
bugfixes:
  - ...
```
Thanks :)

Reply:
Done:)